docs/iris/src/whatsnew/contributions_1.11/bugfix_2016-Oct-06_is_circular.txt

```diff
@@ -0,0 +1,2 @@
+- Fixed a bug where the :func:`iris.util._is_circular` would return false where 
```

Comment:
@marqh LGTM ... but @ajdawson is right, after a quick scan through the last few `whatsnew` all points are marked with `*` ...

... does Sphinx render the html differently with `-` instead of `*` ... I can't tell quickly from documentation, and I can't be bother building the docs to see.  

http://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#list-and-bullets doesn't mention the use of `-` 

@marqh could I nudge you to in the direction of using `*` ? Then I'll merge ... I'll even update the dev notes :wink: